components/dashboard/src/projects/Project.tsx

```diff
@@ -19,6 +19,7 @@ import { getGitpodService, gitpodHostUrl } from "../service/service";
 import { StartWorkspaceModalContext } from "../workspaces/start-workspace-modal-context";
 import { prebuildStatusIcon, prebuildStatusLabel } from "./Prebuilds";
 import { useCurrentProject } from "./project-context";
+import { getProjectTabs } from "./projects.routes";
 import { shortCommitMessage, toRemoteURL } from "./render-utils";
 
 export default function () {
```

Comment:
**issue**: Probably not the best line to comment about this, but now with the project header acting as the breadcrumb, changing org when viewing a project or project settings feels inaccurate. Maybe we could redirect back to the org workspaces list as a home page?

| Project, navigating from Org 1 | Project, after changing org selector |
|-|-|
| <img width="1440" alt="Screenshot 2023-02-02 at 5 47 19 PM (2)" src="https://user-images.githubusercontent.com/120486/216373272-47e6f474-e6ff-4783-bad2-aa992473af30.png"> | <img width="1440" alt="Screenshot 2023-02-02 at 5 47 16 PM (2)" src="https://user-images.githubusercontent.com/120486/216373276-2c3e299b-03f2-4ede-a8d0-d0819560d53b.png"> |

Reply:
It should redirect to / if no such project exists.

Reply:
Opened follow-up https://github.com/gitpod-io/gitpod/issues/16243 to keep track of this. Cc @svenefftinge